scripts/assign.py

```
@@ -207,34 +207,110 @@ def main():
-                        issues_without_prs.append(assigned_issue.get("number"))
+                        graphql_headers = headers.copy()
+                        graphql_headers["Accept"] = "application/vnd.github.v4+json"
+                        graphql_url = "https://api.github.com/graphql"
 
-                if issues_without_prs:
-                    # User has uncompleted issues
-                    issues_list = ", #".join(str(num) for num in issues_without_prs)
+                        variables = {"owner": owner, "repo": repo, "issue_number": current_issue_number}
+
+                        print(f"Checking for linked PRs via GraphQL for issue #{current_issue_number}")
+                        graphql_response = requests.post(
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)